staking/apr/compute.go

```diff
@@ -22,11 +23,11 @@ type Reader interface {
 }
 
 const (
-	nanoSecondsInYear = time.Nanosecond * 3.154e+16
+	secondsInYear = time.Second * 3.154e+7
```

Comment:
you should double check when you make the changes. Underlying, It's the same number Int counting nanoseconds. We don't need the time. package. Just use int64 is fine.

Reply:
You mean just do int64(3.154e+7) directly? (I like this way because it is semantically clear in intent, I liked named constants)

Reply:
yea. because time.Second = 1000000000 in int (it counts in nano seconds in a second). secondsInYear is not actually in seconds, it's still in nano seconds.